bluepyemodel/access_point/local.py

```diff
@@ -158,16 +158,21 @@ def morph_path(self):
             morph_file = recipes["morphology"]
         else:
             for _, morph_file in recipes["morphology"]:
-                if morph_file.endswith(SUPPORTED_MORPHOLOGY_EXTENSIONS):
+                if morph_file.lower().endswith(SUPPORTED_MORPHOLOGY_EXTENSIONS):
                     break
 
-        if not morph_file or not morph_file.endswith(SUPPORTED_MORPHOLOGY_EXTENSIONS):
+        if not morph_file or not morph_file.lower().endswith(SUPPORTED_MORPHOLOGY_EXTENSIONS):
             raise FileNotFoundError(f"Morphology file not defined or not supported: {morph_file}")
 
-        morph_path = self.morph_dir / morph_file
+        matching_files = [
```

Comment:
Do we expect a mismatch between what we have in recipes and the actaul file we have locally?

Reply:
No. We shouldn't allow different morphology than in recipe even if the morphology extension is of different letter case. But we can add to SUPPORTED_MORPHOLOGY_EXTENSIONS the uppercase extensions and you can remove the lower() @ilkilic .

Reply:
maybe adding the uppercase extensions to the this set would be good because we want to match the exact name in the recipes to the filename. Currently, BPEM was giving an error e.g. with ".ASC" format.

`SUPPORTED_MORPHOLOGY_EXTENSIONS = (".asc", ".swc", ".ASC", ".SWC")`